cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -42,6 +42,8 @@
     /{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libudev.so* mr,
     /{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libseccomp.so* mr,
     /{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libcap.so* mr,
+    # TLS padding hack for Android 9
+    /{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libtls-padding.so mr,
```

Comment:
It seems like this should be removed given the prior discussions that it appears unneeded.